src/plugin.tsx

```diff
@@ -157,15 +203,14 @@ export class TranscriptPlugin implements OnMediaUnload, OnMediaLoad, OnPluginSet
     };
 
     private _loadCaptions = (e?: any): void => {
```

Comment:
must we use any here?

Reply:
I haven't found type of this event, implementation taken from:
https://github.com/kaltura/kaltura-interactive-player/blob/ec5a6bd576e1b0fc8542e2a77e0ffd32046efe8d/src/PlayersManager.ts#L458